src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
```

Comment:
Ensure that the file './utils/variables' exists and contains all the necessary variable definitions used across your SCSS files. Missing variables can lead to compilation errors.